### PR TITLE
polish doc in 2.2.x-governance

### PR DESCRIPTION
**Pull request by 123liuziming**
### Describe what this PR does / why we need it
Polish the document in 2.2.x-governance.

### CODE REVIEW

File: spring-cloud-alibaba-examples/governance-example/label-routing-example/readme-zh.md [hunk 1]
```diff
@@ -28,130 +29,139 @@
...
+}
+```
 代码对应的规则如下：
 若同时满足请求参数中含有tag=gray，请求头中含有id且值小于10，uri为/router-test，则50%流量路由到v2版本中，剩下的流量路由到v1版本中，若有一条不满足，则流量路由到v1版本中。
 
 ##### 演示步骤
 1. 访问 http://localhost:18083/add 将路由规则由控制面接口推入路由规则仓库中。
    访问 http://localhost:18083/router-test 不满足路由规则，路由到v1版本中，v1版本实例打印返回如下结果：
+   ```
    NacosRegistration{nacosDiscoveryProperties=NacosDiscoveryProperties{serverAddr='127.0.0.1:8848', endpoint='', namespace='', watchDelay=30000, logName='', service='service-provider', weight=1.0, clusterName='DEFAULT', group='DEFAULT_GROUP', namingLoadCacheAtStart='false', metadata={preserved.register.source=SPRING_CLOUD, version=v1}, registerEnabled=true, ip='XXX', networkInterface='', port=18082, secure=false, accessKey='', secretKey='', heartBeatInterval=null, heartBeatTimeout=null, ipDeleteTimeout=null, failFast=true}}
```
Comment (steverao) [inline spring-cloud-alibaba-examples/governance-example/label-routing-example/readme-zh.md]: The return information is wrong!

File: spring-cloud-alibaba-examples/governance-example/authentication-example/readme-zh.md [hunk 2]
```diff
@@ -6,23 +6,23 @@
...
-1. 修改pom.xml文件，引入Istio资源转换以及Spring Cloud Alibaba鉴权模块:
+1. 修改`pom.xml`文件，引入Istio资源转换以及Spring Cloud Alibaba鉴权模块:
 
 ```xml
 <dependency>
-	<groupId>com.alibaba.cloud</groupId>
-	<artifactId>spring-cloud-starter-alibaba-governance-authentication</artifactId>
+    <groupId>com.alibaba.cloud</groupId>
+    <artifactId>spring-cloud-starter-alibaba-governance-authentication</artifactId>
 </dependency>
```
Comment (steverao) [inline spring-cloud-alibaba-examples/governance-example/authentication-example/readme-zh.md]: Please delete the blank line.

File: spring-cloud-alibaba-examples/governance-example/label-routing-example/readme-zh.md (hunk 1 above)
Comment (123liuziming) [reply spring-cloud-alibaba-examples/governance-example/label-routing-example/readme-zh.md]: done

File: spring-cloud-alibaba-examples/governance-example/authentication-example/readme-zh.md (hunk 2 above)
Comment (123liuziming) [reply spring-cloud-alibaba-examples/governance-example/authentication-example/readme-zh.md]: done

**Review by steverao — APPROVED** [review]

**Pull request merged**